dbt/models/reporting/docs.md

```diff
@@ -32,14 +32,54 @@ Feeds public reporting assets.
 # res_report_summary
 
 {% docs table_res_report_summary %}
-Materialized version of `reporting.vw_res_report_summary`.
+Aggregates statistics on characteristics, classes, AVs, and sales
+by assessment stage, property groups, year, and various geographies.
+Feeds public reporting assets.
 
-Materialized to speed up queries for Tableau.
+Materialized once per day to speed up queries for Tableau.
+
+## Nuance
+
+- Model and assessment values are gathered independently and
+  aggregated via a union rather than a join, so it's important to keep in mind
+  that years for model and assessment stages do NOT need to match, i.e. we can
+  have 2023 model values in the table before there are any 2023 assessment
+  values to report on. Sales are added via a lagged join, so `sale_year` should
+  always be `year - 1`. It is also worth nothing that "model year" is
+  incremented by 1 solely for the sake of reporting in this table, meaning that
+  models with a `meta_year` value of 2022 in `model.assessment_pin` will
+  populate the table with a value of 2023 for `year`.
```

Comment:
This nuance comes from the model docstring. I think it's useful, so I copied it into the dbt docs as well.